ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/jboss/as/jaxrs/main/module.xml

```diff
@@ -41,19 +41,8 @@
         <module name="org.jboss.msc"/>
         <module name="org.jboss.logging"/>
         <module name="org.jboss.logmanager"/>
-        <module name="org.jboss.resteasy.resteasy-atom-provider" optional="true"/>
-        <module name="org.jboss.resteasy.resteasy-cdi" optional="true"/>
-        <module name="org.jboss.resteasy.resteasy-crypto" optional="true"/>
-        <module name="org.jboss.resteasy.resteasy-client"/>
-        <module name="org.jboss.resteasy.resteasy-client-api"/>
         <module name="org.jboss.resteasy.resteasy-core"/>
         <module name="org.jboss.resteasy.resteasy-core-spi"/>
-        <module name="org.jboss.resteasy.resteasy-jaxb-provider" optional="true"/>
-        <module name="org.jboss.resteasy.resteasy-jackson2-provider" optional="true"/>
-        <module name="org.jboss.resteasy.resteasy-jsapi" optional="true"/>
-        <module name="org.jboss.resteasy.resteasy-multipart-provider" optional="true"/>
-        <module name="org.jboss.resteasy.resteasy-spring" optional="true" />
-        <module name="org.jboss.resteasy.resteasy-validator-provider" optional="true"/>
```

Comment:
The subsystem doesn't actually need these modules, but it does add them to deployments in a DUP. I wasn't sure if we needed to keep them as optional dependencies or not based on that.

Reply:
@jamezp I think ResourceDefinition.registerAdditionalRuntimePackages() is intended for this. I see I am using that for Reactive Messaging

Reply:
Yes, sorry I missed that. I wasn't sure if it's better to keep the modules here or to add them in the `ResourceDefinition`. The `ResourceDefinition` seems more in the spirit of JBoss Modules and Galleon.

Reply:
I've just reverted the `module.xml` back for now. A follow up can be done later.